styles.css

```diff
@@ -24,8 +30,55 @@ body {
   margin: 12px 8px;
 }
 
+.menu {
+  margin: 0;
+  position: fixed;
+  transform: translateY(-100%);
+  transition: transform 0.2s;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #6070ff;
+  mix-blend-mode: multiply;
+  color: white;
+  list-style: none;
+  padding-top: 4rem;
+  align-items: center;
+  text-align: center;
+  text-decoration: none;
+  opacity: 0.8;
+  backdrop-filter: blur(80px);
+}
```

Comment:
- [ ] Kindly make sure your design to match  Figma design by using the correct Hemberger and close icon and also make sure to add blur effect to mobile menu to match the design as figma.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/175313438-e8572f8c-1e4c-44f5-82ee-2e90e22b82c1.png)  | ![image](https://user-images.githubusercontent.com/53402600/175313377-2bba5113-4181-4d16-a96c-e69eb681ce70.png) |